docs/code-snippets/word-snippets.yaml

```diff
@@ -1,3 +1,19 @@
+Word.Application#createDocument:member(1):
+  - |-
+    // Update the text of the current document using the text from another document passed in as base64.
+    await Word.run(async (context) => {
+        const externalDoc = context.application.createDocument(getBase64());
```

Comment:
Where does `getBase64` come from? That feels like an important step.

Reply:
Asked code source for this and now waiting for their response

Reply:
In the meantime, added a comment about need to implement getBase64 that's similar to the comment in [Word.Body.insertFileFromBase64](https://learn.microsoft.com/en-US/javascript/api/word/word.body?view=word-js-preview#word-word-body-insertfilefrombase64-member(1))